src/Microsoft.DotNet.Build.CloudTestTasks/DownloadFromAzure.cs

```diff
@@ -38,9 +38,20 @@ public sealed class DownloadFromAzure : AzureConnectionStringBuildTask
 
         public int MaxClients { get; set; } = 8;
 
+        [Output]
+        public ITaskItem[] PathTooLongBlobs
+        {
+            get
+            {
+                return pathTooLongEntries.ToArray();
+            }
+        }
+
         private static readonly CancellationTokenSource TokenSource = new CancellationTokenSource();
         private static readonly CancellationToken CancellationToken = TokenSource.Token;
 
+        private IList<ITaskItem> pathTooLongEntries = new List<ITaskItem>();
```

Comment:
Private variables should be prefixed with `_`

Reply:
Thanks! I was wondering for the convention here. The different projects use different conventions, some use m_ some have nothing. BTW Looking for a guidance I just noticed that the Contributing doc seems broken for this particular repo.